File: Samples/C#.Net/CSharpSDKTest/ISmpUserDlg.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using SalesMatePlusLib;

namespace CSharpSDKTest
{
    public partial class ISmpUserDlg : Form
    {
        public ISmpUserDlg()
        {
            InitializeComponent();
        }

        private void loginbtn_Click(object sender, EventArgs e)
        {
            ISmpUser m_SmpUser;

            m_SmpUser = new SmpUserClass();

            m_SmpUser.InvokeLoginUserDlg();
        }

        private void button2_Click(object sender, EventArgs e)
        {
            ISmpUser m_SmpUser;

            m_SmpUser = new SmpUserClass();

            m_SmpUser.InvokeChangePasswordDlg();
        }
    }
}

File: Samples/VB.Net/VBDotNet/ISmpUserDlg.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports SalesMatePlusLib

Namespace CSharpSDKTest
	Partial Public Class ISmpUserDlg
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub loginbtn_Click(ByVal sender As Object, ByVal e As EventArgs) Handles loginbtn.Click
			Dim m_SmpUser As ISmpUser

			m_SmpUser = New SmpUserClass()

			m_SmpUser.InvokeLoginUserDlg()
		End Sub

		Private Sub button2_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button2.Click
			Dim m_SmpUser As ISmpUser

			m_SmpUser = New SmpUserClass()

			m_SmpUser.InvokeChangePasswordDlg()
		End Sub
	End Class
End Namespace